### DISCUSSION TITLE
website/posts/20240601_yagi3/

### DISCUSSION THREAD

**Discussion by giscus** [roninro/hugo-theme-puppet#72] [General]
# website/posts/20240601_yagi3/

yagi3です。 本日の内容今日はアイズについて学びました。 https://onlinejudge.u-aizu.ac.jp/home おもしろそうな課題が多そうでしたので、 今後やっていきたいです。 次回はｍ６月１５日

https://ichikawapc.github.io/website/posts/20240601_yagi3/

**Comment by mavecafe** [comment]
そういえば、学長が６月１日はリベシティ４周年（２０２０年から）といっていたな！
